test/Microsoft.ML.Predictor.Tests/TestAutoInference.cs

```diff
@@ -20,14 +20,14 @@ public TestAutoInference(ITestOutputHelper helper)
         {
         }
 
-        [Fact(Skip = "Need CoreTLC specific baseline update")]
+        [Fact]
         [TestCategory("EntryPoints")]
         public void TestLearn()
         {
             using (var env = new TlcEnvironment())
             {
-                string pathData = GetDataPath(@"../UCI/adult.train");
-                string pathDataTest = GetDataPath(@"../UCI/adult.test");
+                string pathData = GetDataPath(@"../data/adult.train");
```

Comment:
>/data/adult [](start = 50, length = 11)

[https://github.com/dotnet/machinelearning/tree/master/Samples/UCI](https://github.com/dotnet/machinelearning/tree/master/Samples/UCI) we have them our repository, no need to add them to another folder

Reply:
Thanks. I didn't realize we had them, since there was an adult.tiny dataset in the folder I had added them to, but no adult train and test. Thanks for the heads up, I'll update the PR.